contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -11,14 +11,23 @@ toc::[]
 == Modularization backstory
 OpenShift docs are getting modularized, starting from OpenShift 4.0.
 All existing content will be completely replaced with content that is based on user stories and
-complies with the modularization guidelines. All future content must support a user story.
+complies with the modularization guidelines. All future content must both
+support a user story and be modular.
 
-== How do I contribute user stories?
-Familiarize yourself with the Customer Content Services (CCS)
-link:https://redhat-documentation.github.io/modular-docs/[modularization guide]. This is an important
-first step in making sure you are helping the docs team create content that is fully modularized.
+== How do I contribute modularized content?
+To contribute modularized content, you need to write a user story, create
+documentation modules to support the user story, and create an assembly for the
```

Comment:
Maybe links to the _How do I write in modules?_ section or a brief description of modules and assemblies? Again, to show that we will tell the reader what these things are.